runtime/java/build.xml

```diff
@@ -26,7 +26,7 @@
   </target>
 
   <target name='build' depends='init'>
-    <javac debug='on' classpathref='compile.classpath' srcdir='${src}:${grammars}' destdir='${bin}' includeantruntime='false' source='1.8' target='1.8' release='8'>
+    <javac debug='on' classpathref='compile.classpath' srcdir='${src}:${grammars}' destdir='${bin}' includeantruntime='false' release='11'>
```

Comment:
Are we intentionally bumping our minimum JVM version?

Reply:
Yeah, I don't remember exactly but I think I needed some file API for examining jars that was added in Java 11. It was released in 2018 and Java 8 is EOL in a couple of years so I don't think this is going to be an issue for anyone.

Reply:
okay, we probably need to update at least the readme then; any other docs that say jdk/jvm/java 8 too

Reply:
Updated the README; I didn't see any other references to Java 8 in the docs.